yongkeun/week3/과제4/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Mochiy+Pop+One&display=swap" rel="stylesheet">
+    <title>Cards</title>
+</head>
+<body>
+    <article>
+        <div class="flip">
+            <div class="card">
+            <div id="first-card" class="front"><h1>Financial App</h1>25 Apr 2030</div>
+            <div id="first-card" class="back">Click!</div>
```

Comment:
하나의 id는 html에서 한 번만 사용해야 해요!
id로 자바스크립트에 접근하는 방법은 좋았으나 다른 방법을 생각해봐요!
```suggestion
            <div id="first-card" class="front"><h1>Financial App</h1>25 Apr 2030</div>
            <div class="back">Click!</div>
```